botbot/apps/plugins/core/logger.py

```diff
@@ -8,9 +8,9 @@
 
 class Config(config.BaseConfig):
     ignore_prefix = config.Field(
-        default="!-",
+        default=["!-"],
         required=False,
-        help_text="Don't log lines starting with this string"
+        help_text="Don't log lines starting with any strings in this list"
```

Comment:
You need to be explicit that this is a regular expression now, since `/[off]/` means something very different to `"[off]"`